tests/test_project.py

```diff
@@ -50,6 +55,11 @@ def test_dataset_load():
     assert "subsector" in lookup.columns
     assert "id" in lookup.columns
 
+    query_names = sorted(project.config.list_dimension_query_names(DimensionType.GEOGRAPHY))
+    assert query_names == ["census_division", "census_region", "county", "state"]
+    records = project.config.get_dimension_records(DimensionType.GEOGRAPHY, "state")
+    assert records.filter("id = 'CO'").count() > 0
+
```

Comment:
Add check for "subsectors", "all_subsectors" query_names?

Reply:
Is this is a stale comment?